### PR TITLE
Review and Adaptation to PR #9864

### PR DESCRIPTION
**Pull request by gildegoma**

### CODE REVIEW

**Comment by gildegoma** [comment]
@briancain @chrisroberts Pulling a reviewer for english quality seal :blush:

**Comment by rgarrigue** [comment]
@gildegoma :+1: for the rewriting, indeed it's more compliant way

File: website/source/docs/provisioning/ansible_local.html.md [hunk 1]
```diff
@@ -143,6 +143,29 @@ This section lists the _specific_ options for the Ansible Local provisioner. In
 
 ## Tips and Tricks
 
+### Install Galaxy Roles in a path owned by root
+
+ <div class="alert alert-warning">
+    <strong>Disclaimer:</strong> This tip is not a recommendation to install galaxy roles out of the vagrant user space, especially if you rely on ssh agent forwarding to fetch the roles.
+</div>
+
+Be careful that `ansible-galaxy` command is executed by default as vagrant user. Setting `galaxy_roles_path` to a folder like `/etc/ansible/roles` will fail, and `ansible-galaxy` will extract the role a second time in a `/home/vagrant/.ansible/roles/`. Then if your playbook uses `become` to run as `root`, it will fail with a _"role was not found"_ error.
```
Comment (briancain) [inline website/source/docs/provisioning/ansible_local.html.md]: Small wording nit pick, but I think `a` should be removed from this sentence...

>and `ansible-galaxy` will extract the role a second time in a `/home/vagrant/.ansible/roles/`.

to

>and `ansible-galaxy` will extract the role a second time in `/home/vagrant/.ansible/roles/`.

**Comment by briancain** [comment]
Just a small bit of feedback, otherwise looks good to me! 👍

File: website/source/docs/provisioning/ansible_local.html.md (hunk 1 above)
Comment (gildegoma) [reply website/source/docs/provisioning/ansible_local.html.md]: @briancain thank you very much for the good catch, and taking time to review this :heavy_heart_exclamation:

**Comment by gildegoma** [comment]
@briancain I think this should be all set for the merge.

**Comment by briancain** [comment]
Thanks! These changes should be put in place the next time we deploy the website.

**Pull request merged**